src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/seed/SeedFromXLSXApprovedPremisesRoomsTest.kt

```diff
@@ -2,23 +2,27 @@ package uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.cas1.seed
 
 import org.assertj.core.api.Assertions.assertThat
 import org.jetbrains.kotlinx.dataframe.api.dataFrameOf
+import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.TestInstance
-import org.springframework.beans.factory.annotation.Autowired
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.SeedFromExcelFileType
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.seed.SeedTestBase
-import uk.gov.justice.digital.hmpps.approvedpremisesapi.seed.cas1.SiteSurvey
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.seed.cas1.QuestionCriteriaMapping
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.seed.cas1.SiteSurveyImportException
 import java.util.UUID
 
 @TestInstance(TestInstance.Lifecycle.PER_METHOD)
 class SeedFromXLSXApprovedPremisesRoomsTest : SeedTestBase() {
 
-  @Autowired
-  lateinit var siteSurvey: SiteSurvey
+  private lateinit var questionCriteriaMapping: QuestionCriteriaMapping
+
+  @BeforeEach
+  fun setup() {
+    questionCriteriaMapping = QuestionCriteriaMapping(characteristicRepository)
+  }
 
   fun MutableList<String>.addCharacteristics(numberOfRooms: Int = 1, activeCharacteristics: Map<String, List<Int>> = emptyMap()) {
-    siteSurvey.questionToCharacterEntityMapping.keys.forEach { question ->
+    questionCriteriaMapping.questionToCharacterEntityMapping.keys.forEach { question ->
```

Comment:
as discussed, propose copying all questions into a List<String> in this class as an instance variable, and then referring to that here instead of `questionCriteriaMapping.questionToCharacterEntityMapping.keys`

Reply:
Will revisit this when work is done to address [APS-1757](https://dsdmoj.atlassian.net/browse/APS-1757).